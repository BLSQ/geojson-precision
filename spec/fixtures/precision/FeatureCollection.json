{
    "type": "FeatureCollection",
    "features": [
      {
        "type": "Feature",
        "properties": {},
        "geometry": {
          "type": "Polygon",
          "coordinates": [
            [
              [
                -69.60937499999999,
                51.17934297928927
              ],
              [
                -71.015625,
                48.45835188280866
              ],
              [
                -93.8671875,
                28.613459424004414
              ],
              [
                -45,
                24.206889622398023
              ],
              [
                -39.7265625,
                41.244772343082076
              ],
              [
                -46.7578125,
                47.98992166741417
              ],
              [
                -69.60937499999999,
                51.17934297928927
              ]
            ]
          ]
        }
      },
      {
        "type": "Feature",
        "properties": {},
        "geometry": {
          "type": "Polygon",
          "coordinates": [
            [
              [
                -31.289062500000004,
                10.833305983642491
              ],
              [
                -31.289062500000004,
                34.30714385628804
              ],
              [
                -7.03125,
                34.30714385628804
              ],
              [
                -7.03125,
                10.833305983642491
              ],
              [
                -31.289062500000004,
                10.833305983642491
              ]
            ]
          ]
        }
      },
      {
        "type": "Feature",
        "properties": {},
        "geometry": {
          "type": "Point",
          "coordinates": [
            18.984375,
            57.32652122521709
          ]
        }
      },
      {
        "type": "Feature",
        "properties": {},
        "geometry": {
          "type": "LineString",
          "coordinates": [
            [
              19.6875,
              33.43144133557529
            ],
            [
              11.25,
              45.089035564831015
            ],
            [
              -7.03125,
              46.558860303117164
            ],
            [
              -11.6015625,
              54.367758524068385
            ],
            [
              6.328125,
              53.5403073915002
            ],
            [
              35.15625,
              51.17934297928927
            ]
          ]
        }
      }
    ]
  }