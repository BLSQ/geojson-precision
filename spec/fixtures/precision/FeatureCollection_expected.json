{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {},
      "geometry": {
        "type": "Polygon",
        "coordinates": [
          [
            [
              -69.609,
              51.179
            ],
            [
              -71.016,
              48.458
            ],
            [
              -93.867,
              28.613
            ],
            [
              -45.0,
              24.207
            ],
            [
              -39.727,
              41.245
            ],
            [
              -46.758,
              47.99
            ],
            [
              -69.609,
              51.179
            ]
          ]
        ]
      }
    },
    {
      "type": "Feature",
      "properties": {},
      "geometry": {
        "type": "Polygon",
        "coordinates": [
          [
            [
              -31.289,
              10.833
            ],
            [
              -31.289,
              34.307
            ],
            [
              -7.031,
              34.307
            ],
            [
              -7.031,
              10.833
            ],
            [
              -31.289,
              10.833
            ]
          ]
        ]
      }
    },
    {
      "type": "Feature",
      "properties": {},
      "geometry": {
        "type": "Point",
        "coordinates": [
          18.984,
          57.327
        ]
      }
    },
    {
      "type": "Feature",
      "properties": {},
      "geometry": {
        "type": "LineString",
        "coordinates": [
          [
            19.688,
            33.431
          ],
          [
            11.25,
            45.089
          ],
          [
            -7.031,
            46.559
          ],
          [
            -11.602,
            54.368
          ],
          [
            6.328,
            53.54
          ],
          [
            35.156,
            51.179
          ]
        ]
      }
    }
  ]
}